trusted-committer/de/07-becoming-a-trusted-committer.asciidoc

```diff
@@ -0,0 +1,22 @@
+
+Der Prozess, offiziell ein _Trusted Committer_ zu werden, unterscheidet sich von Community zu Community und hängt davon ab, wo du persönlich in deiner Innner Source Reise stehst, und wird sich im Lauf der Zeit entwickeln. In neugegründeten Communities übernehmen die Gründer oft selbst die Rolle des _Trusted Committers_. Wenn die Community dann wächst, oder in größeren Communities, werden _Trusted Committer_ normalerweise nominiert oder von den https://innersourcecommons.org/resources/learningpath/contributor/index[Contributors] der Community gewählt.
+Trotzdem sollte die Rolle des _Trusted Committers_ freiwillig übernommen werden, denn sie erfordert um erfolgreich zu sein, großen Zeitaufwand und Engagement. 
+
+Welche Kriterien gelten bei der Nominierungen von https://innersourcecommons.org/resources/learningpath/contributor/inde[Contributors] für die Rolle des _Trusted Committers_? Was benötigt man, um die Rolle des _Trusted Committers_ erfolgreich auszufüllen? Zuallererst müssen potentielle _Trusted Committer_ ihre tiefe technische Kompetenz während ihrer Arbeit in der Community unter Beweis stellen. Zusätzlich dazu müssen sie ihre Fähigkeit unter Beweis stellen, wirkunsvoll mit den Kollegen der Community zusammenzuarbeiten, und idealerweise genauso mit den Product Ownern und dem Management.
+
+Auf die selbe Art müssen sie den Willen und die Geduld aufweisen, ihre Fähigkeiten zu nutzen und bewusst ihre Zeit spendieren, um _Contributors_ weiter zu qualifizieren. Schließlich erfordert die Erfüllung der Rolle des _Trusted Committers_ eine gewisse emotinale Reife, um mit stressigen sozialen Situationen umzugehen, die mit Sicherheit ab und zu aufkommen.
+Contributors, welche diese Kritierien erfüllen, sind nach unserer Meinung gute Kandidaten für potentielle _Trusted Committers_.
+
+Die Rolle des _Trusted Committers_ wird nicht für alle Contributors attraktiv sein, bedeutet es doch, dass man weniger Zeit verbringt, zu selbst zu programmieren. Als _Trusted Committer_ vorgeschlagen zu werden kann sogar herabstufend empfunden werden oder als negatives Feedback bezüglich der eigenen Programmierfähgigkeiten aufgefasst werden. Aber das Gegenteil ist der Fall. Als _Trusted Committer_ nominiert zu werden, bedeutet normalerweise, dass man deine wertvollen Beiträge erkannt hat und in dir das Potential weiter zu wachsen und zu führen sieht. Die Rolle des _Trusted Committers_ gib dir mehr Einfluß auf die Entwicklung der Codebasis und macht dich definitiv zu einem vollständigeren Entwickler. Contributors zu erklären, wie die Software funktioniert, führt häufig zu neuen Erkenntnissen seitens des _Trusted Committers_ und hilft Möglichkeiten zu identifizieren, die Software weiter zu verbessern.
+
+Ob es einer InnerSource Community einer oder mehrere _Trusted Committer_ gibt, hängt von der Größe und dem Risiko, welches mit ihrer Software verbunden ist, ab.
```

Comment:
```suggestion
Ob es einer InnerSource Community einen oder mehrere _Trusted Committer_ gibt, hängt von der Größe und dem Risiko, welches mit ihrer Software verbunden ist, ab.
```

Reply:
strange thing, my local and my remote rep do not show the red part, there is already the green one visible?